build/dataHash.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
```

Comment:
The other file names are just a single word, and this one have two, is this a problem?

Reply:
that seems ok